README.md

```diff
@@ -267,6 +267,12 @@ Please note that you can either return a Promise or call the third argument of t
 
 Additional note: If you need to access the sequelize instance, you can easily do that via `queryInterface.sequelize`.
 
+## Custom Seeder and Migration templates
+
+The CLI can use your custom migration/seed templates. Simply create a file called skeleton.js in the seeders/migrations folder created by the init command. 
+
+The skeleton must export both an up and a down function.
```

Comment:
Maybe not a good idea. If someone adds the files there, then they know what they are doing. If we just add the skeletons, someone might do something accidentally.

Maybe having a task that copies the skeletons to the migrations/seeders folder?